lib/shipit.rb

```diff
@@ -110,8 +113,8 @@ def all_settings_present?
     ].all?(&:present?)
   end
 
-  def extra_env
-    secrets.env || {}
+  def env
+    {'SHIPIT' => '1'}.merge(secrets.env || {})
```

Comment:
A lot of this is actually just a great refactoring, would you mind splitting it off?